composer.json

```diff
@@ -40,9 +40,10 @@
     "neos/setup": "@dev",
     "flowpack/neos-frontendlogin": "@dev",
     "mikey179/vfsstream": "^1.6",
-    "phpunit/phpunit": "^9.0",
+    "behat/behat": "^3.0",
+    "phpunit/phpunit": "^10.0",
```

Comment:
dont we have like code that needs to be adjusted? ^^

Reply:
see above